packages/msi/msi/Product.wxs

```diff
@@ -6,13 +6,23 @@
 
 <?define UpgradeCodeStandard = "b0fee933-ccd2-467c-8fe4-bb0ac6a099c8" ?>
 <?define UpgradeCodeEnterprise = "4ABDD5C7-E1E1-41A6-8119-DCE65634A6CC" ?>
-<?if $(var.MondooSKU) = "standard" ?>
+<?define UpgradeCodeArm64 = "090cfb7d-c00c-4d36-94fe-f649a4b29c91" ?>
+<?if $(var.arch) = "arm64"?>
+<?define UpgradeCode = "$(var.UpgradeCodeArm64)"?>
+<?define InstallerVersion="500"?>
+<?define ProductName = "Mondoo"?>
```

Comment:
Why do we need the product name in the if condition?

Reply:
moved all common elements out of the conditional block.